src/cljs/org/broadinstitute/firecloud_ui/utils.cljs

```diff
@@ -93,3 +93,10 @@
     (fn [x1 x2] (if (and (map? x1) (map? x2)) (deep-merge x1 x2) x2))
     maps))
 
+
+(defn generate-form-data [params]
+  "Create a blob of multipart/form-data from the provided map."
```

Comment:
Annoyingly, the docs for `defn` have the doc string directly after the name and before the parameters. We should probably follow this even though it sucks.